src/Genome.cpp

```diff
@@ -114,15 +129,24 @@ double Genome::siteCostScore() {
 	return out;// / maxCost;
 }
 
+
+//Expected: nothing
+//Returns: percentage of mec over maxMEC
+//Question: what is maxMEC?
```

Comment:
good question! I think this maxMec is wrong, becuase it doesn't account for coverage, it's just the number of sites * number of haplotypes. (eg 2 for diploid)